src/main/java/com/palominolabs/metrics/guice/intercept/CountedInterceptor.java

```diff
@@ -1,16 +1,16 @@
-package com.palominolabs.metrics.guice;
+package com.palominolabs.metrics.guice.intercept;
 
 import com.codahale.metrics.Counter;
 import com.codahale.metrics.annotation.Counted;
 import org.aopalliance.intercept.MethodInterceptor;
 import org.aopalliance.intercept.MethodInvocation;
 
-class CountedInterceptor implements MethodInterceptor {
+public class CountedInterceptor implements MethodInterceptor {
```

Comment:
What's your reasoning for making these types public? I'd like to avoid making them part of the public API if possible.